docs/examples/DataSet/Extracting-runs-from-one-DB-file-to-another.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Extracting runs from one DB file to another\n",
     "\n",
-    "This notebook shows how to use the `extract_runs_into_db` function to extract runs from DB file (the source DB) into another DB file (the target DB). If the target DB does not exist, it will be created. The runs are **NOT** removed from the original DB file; they are copied over.\n"
+    "This notebook shows how to use the `extract_runs_into_db` method to extract runs from a database (DB) file (the source DB) into another DB file (the target DB). If the target DB does not exist, it will be created. The runs are **NOT** removed from the original DB file; they are copied over.\n"
```

Comment:
as per python definitions, it's a function, not a method :)
```suggestion
    "This notebook shows how to use the `extract_runs_into_db` function to extract runs from a database (DB) file (the source DB) into another DB file (the target DB). If the target DB does not exist, it will be created. The runs are **NOT** removed from the original DB file; they are copied over.\n"
```

Reply:
:)